code-challenges.js

```diff
@@ -6,44 +6,171 @@
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+// Declared a varibale that will group all the temps
+// let allTemp = (temp) => {
+// use conditional statments to compare. use strictly equals for the at boiling point
+//    if(temp === 212) {
+    // if the temp strictly equals 212 then return is at boiling point
+//        return `${temp} is at boiling point`
+// use an else if statment for when the temp is less than 212 and if it is, return the temp and say it is below boiling point
+//    } else  if(temp < 212){
+//        return `${temp} is below boiling point`
+// use else as a catch all for anything else and return is above boilin point
+//    }  else {
+//        return `${temp} is above boiling point`
+//   } 
+// }
+// finally, invoke the functions
+// console.log(allTemp(temp1))
+// console.log(allTemp(temp2))
+// console.log(allTemp(temp3))
+
+
+
+
+
+
+
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
+//create a function named allNumbers 
+// use concatenation to add the two arrays
+// let allNumbers = myNumbers1.concat(myNumbers2)
+// invoke the function with .length to get the length of the new array
+// console.log(allNumbers.length)
+
+
+
+
+
+
+
+
+
+
+
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 ovarB"
+
+// const currentCohort = "Bravo 2022"
+// Declare a variable to split the string and use .split
+// let cohortSplit = currentCohort.split("")
+// Declare a variable to reverse the new split string using .reverse
+// let cohortReverse = cohortSplit.reverse()
+// Declare a new variable to join the reversed string using .join
+// let cohortJoin = cohortReverse.join("")
+// invoke the functions
+// console.log(cohortSplit)
+// console.log(cohortReverse)
+// console.log(cohortJoin)
+
+
+
+
+
+
+
+
+
+
 
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
-const currentCohort = "Bravo 2022"
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-const myArray = [13, 34, 5, 10, 27, 42]
+// const myArray = [13, 34, 5, 10, 27, 42]
+// Create a for-loop that loops through until the end of the array length
+// for(let i =0; i < myArray.length; i++){
+// use modulo to determine which numbers are odd and even
+// if(myArray[i] % 2 !== 0){
+// console.log(`${myArray[i]} odd`);
+//   } else{
+    //invoke the fuction
```

Comment:
This is actually not a function.  The console log on 139 is displaying your result to the terminal.  A function would have arrow notation and allow us to pass an argument through the invocation and use a parameter in the function to access the data.